transactions/views.py

```diff
@@ -427,17 +427,16 @@ def tokenized_card_payment(request):
     user = request.user
     serializer = CardlessPaymentSerializer(data=data)
     if serializer.is_valid():
+        card = get_object_or_404(CardInfo, id=kwargs.get('saved_card_id'))
```

Comment:
This works just fine, but Ive noticed the error message that is returned here,might not be something that the user might understand 

```json
{"errors":"No CardInfo matches the given query."}
```
since we are to show the users the different errors that our backend is returning, what do you think about creating a custom error message , that may mean we change how a card is being retrieved to something like

```py
try:
    card = CardInfo.objects.get(id=kwargs.get('saved_card_id'))
except CardInfo.DoesNotExist:
    return Response({'errors': 'The card you specified does not exist, try a different card'}, status=status.HTTP_404_NOT_FOUND)
```

Reply:
That's a great suggestion, I worked on this.